RadixWallet/Clients/OnLedgerEntitiesClient/OnLedgerEntitiesClient+Live.swift

```diff
@@ -2,7 +2,7 @@ import Sargon
 
 // MARK: - OnLedgerEntitiesClient + DependencyKey
 extension OnLedgerEntitiesClient: DependencyKey {
-	public static let maximumNFTIDChunkSize = 29
+	public static let maximumNFTIDChunkSize = 100
```

Comment:
So you are saying that GW might not return pages with the max number of items, even if there are more pages available? Didn't quite understand why this value should change.

Reply:
There are two things here:
- This number changes to 100 because such is the value that each page can return now (and not 29 as it previously did).
- The GW will change to potentially return less items than what it fit on one page, even if there are items to return. For example, if there are 250 items, it may return `[100, 80, 70]` and not `[100, 100, 50]` (which is what the app, without my change, currently expects and why it wouldn't work if different amount of items were returned).

Reply:
Got it, thanks for the clarification